modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TreeTableRowSkinTest.java

```diff
@@ -252,6 +251,53 @@ public void treeTableRowWithFixedCellSizeShouldIgnoreVerticalPadding() {
         assertEquals(height, cell.getHeight(), 0);
     }
 
+    /**
+     * When we set a fixed cell size and make an invisible column visible we expect the underlying cells to be visible,
+     * e.g. width > 0.
+     * See also: JDK-8305248
+     */
+    @Test
+    public void testMakeInvisibleColumnVisible() {
```

Comment:
These two tests also pass with/without your patch, but I guess we want to have them to prevent any future issue?

Reply:
Yes. `TreeTableView` does a lot more layout passes then `TableView`. 
I had a look on this in the past and in general the table implementations do too much (re)layouting unfortunately, and that sometimes masks bugs, like this one.